build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,128 @@
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibHostFilesUnix)"
+        DestinationFolder="$(XAFrameworkDir)/lib"
+    />
+    <Copy
+        SourceFiles="@(VersionFiles)"
+        DestinationFolder="$(XAFrameworkDir)"
+    />
+    <Copy
+        SourceFiles="@(TargetsFiles)"
+        DestinationFolder="$(XAFrameworkDir)/lib/xamarin.android/xbuild/%(RecursiveDir)"
+    />
+  </Target>
+  <Target Name="_CreateSymbolicLinks"
+      DependsOnTargets="_CopyFilesToPayloadDir">
+    <Exec WorkingDirectory="$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions"
+      Command="ln -fs $(XAVersion) Current"
+    />
+    <Exec WorkingDirectory="$(MSBuildTargetsDir)/lib"
+      Command="ln -fs host-$(HostOS) host"
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild"
+      Command="ln -fs ../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/xamarin.android/xbuild/Novell ."
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild/Xamarin"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild/Xamarin"
+      Command="ln -fs ../../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/xamarin.android/xbuild/Xamarin/Android ."
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild-frameworks"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild-frameworks"
+      Command="ln -fs ../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/xamarin.android/xbuild-frameworks/MonoAndroid ."
+    />
+  </Target>
+  <Target Name="_FinalizePayload"
+      DependsOnTargets="_CreateSymbolicLinks">
+    <ReplaceFileContents
+        SourceFile="distribution.xml.in"
+        DestinationFile="$(PkgDistributionDestination)"
+        Replacements="@PACKAGE_TITLE@=Xamarin.Android $(XAVersion)">
+    </ReplaceFileContents>
+    <Exec
+        WorkingDirectory="$(XamarinAndroidSourcePath)"
+        ConsoleToMSBuild="true"
+        Command="git log --no-color --first-parent -n1 --pretty=format:%ct">
+      <Output TaskParameter="ConsoleOutput" PropertyName="UpdateInfoVersion"/>
+    </Exec>
+    <WriteLinesToFile
+        File="$(XAFrameworkDir)/updateinfo"
+        Lines="$(UpdateInfoGuid) $(UpdateInfoVersion)"
+        Overwrite="true"
+    />
+    <MakeDir Directories="$(PkgLicenseDestinationEn)"/>
+    <Copy
+        SourceFiles="$(PkgLicenseSrcEn)"
+        DestinationFiles="$(PkgLicenseDestinationEn)/License"
+    />
+  </Target>
+  <Target Name="_PackageArchive"
+      DependsOnTargets="_FinalizePayload">
+    <RemoveDir Directories="$(PkgOutputPath)" />
+    <MakeDir Directories="$(PkgOutputPath)"/>
+    <Exec Command="pkgbuild --root &quot;$(PayloadDir)&quot; --identifier com.xamarin.android.pkg --version $(XAVersion) --install-location &quot;$(PkgInstallDir)&quot; &quot;$(PkgOutputPath)/xamarin.android.pkg&quot;"/>
+  </Target>
+  <Target Name="CreatePkg"
```

Comment:
This target and dependencies should be `Conditional` on `'$(HostOS)' == 'Darwin'`.  We don't want this target to do anything when building on Linux, as it will fail there.